x-pack/plugins/data_usage/public/app/components/data_usage_metrics.tsx

```diff
@@ -33,6 +35,8 @@ export const DataUsageMetrics = memo(
   ({ 'data-test-subj': dataTestSubj = 'data-usage-metrics' }: { 'data-test-subj'?: string }) => {
     const getTestId = useTestIdGenerator(dataTestSubj);
 
+    const [isFirstPageLoad, setIsFirstPageLoad] = useState(true);
```

Comment:
this is to ensure not to overwrite data streams to URL when we click on `clear all` data streams filter